citesphere/src/main/webapp/WEB-INF/views/auth/group/items.html

```diff
@@ -599,6 +600,11 @@ <h4 class="panel-title">
 				</a>
 				</span>
 			</td>
+			<td>
+			    <a class="text-warning" th:href="@{|/auth/group/${zoteroGroupId}/items/${entry.key}?index=${loop.index}&collectionId=${collectionId}&page=${currentPage}&sortBy=${sort}|}">
```

Comment:
using the right class (`bib-entry`) should fix the color issue without using the color attribute.